### PR TITLE
Add the classic Efficient Smalltalk '80 paper.

### PR DESCRIPTION
**Pull request by ltratt**
I'm surprised we didn't have this one in already!

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2524,3 +2524,13 @@ @article{aycock03brief
     doi = {10.1145/857076.857077},
     acmid = {857077},
 }
+
+@inproceedings{deutsch84efficient,
+    author = "L. Peter Deutsch and Allan M. Schiffman",
```
Comment (vext01) [inline softdev.bib]: Should the first author's first name be in full?

Comment (ltratt) [reply softdev.bib]: That's the form he uses on the paper itself, so I think this is correct.

**Comment by vext01** [comment]
OK, if you sync the branch, we can merge.

**Comment by ltratt** [comment]
Synced.

**Pull request merged**